packages/universal-components/src/RangeDatePicker/RangeDatePicker.js

```diff
@@ -76,11 +78,13 @@ export default class RangeDatePicker extends React.Component<Props> {
         <View style={styles.content}>
           {!isNightsInDestinationSelected || !isNightsInDestinationVisible ? (
             <RangeDatePickerContent
+              isChoosingPastDatesEnabled={isChoosingPastDatesEnabled ?? false}
```

Comment:
Why not use default props for that?

Reply:
Because `defaultProps` will not convince flow that it's fine that way 😄. It doesn't work without condition. How should I do it properly without repeating default value on more than one place?

Reply:
BTW: 
If you will be set the `isChoosingPastDatesEnabled` flow type to required, and then set the default props `isChoosingPastDatesEnabled` to false. It will be work, and it will be optional 😄